src/Facility.CodeGen.CSharp/CSharpServiceInfo.cs

```diff
@@ -81,12 +81,36 @@ public static bool TryCreate(ServiceInfo serviceInfo, CSharpServiceInfoSettings?
 
 	internal string GetErrorName(ServiceErrorInfo errorInfo) => FixName(errorInfo.Name);
 
+	internal string GetExternalDtoName(ServiceExternalDtoInfo info)
+	{
+		var typeName = m_externalDtoNames.TryGetValue(info, out var name) ? name : $"{FixName(info.Name)}Dto";
+		var typeNamespace = m_externalDtoNamespaces.TryGetValue(info, out var ns) ? ns : "";
+		if (!string.IsNullOrEmpty(typeNamespace))
```

Comment:
```suggestion
		if (typeNamespace.Length != 0)
```
Nit: don't test for impossible conditions. Also below.

Reply:
Used suggestion below